hbase-thrift/src/test/java/org/apache/hadoop/hbase/thrift/TestThriftServer.java

```diff
@@ -871,4 +878,32 @@ private void closeScanner(
     handler.scannerGet(scannerId);
     handler.scannerClose(scannerId);
   }
+
+  @Test
+  public void testGetThriftServerType() throws Exception {
+    ThriftHBaseServiceHandler handler =
+        new ThriftHBaseServiceHandler(UTIL.getConfiguration(),
+            UserProvider.instantiate(UTIL.getConfiguration()));
+    assertEquals(TThriftServerType.ONE, handler.getThriftServerType());
+  }
+
+  @Test
+  public void testGetThriftServerOneType() throws Exception {
```

Comment:
nit: test method says 'One'Type but the test spawns a server of 'two' type.

Reply:
verify that thrift2 client calling thrift server can get the type correctly. more discussion see:https://issues.apache.org/jira/browse/HBASE-20395

Reply:
I see, thanks for the context. Probably helps to add a quick comment to clear this up?